keys.go

```diff
@@ -362,6 +362,24 @@ func (c *Client) Rotate(ctx context.Context, id, payload string) error {
 	return nil
 }
 
+// Disable a key
+func (c *Client) Disable(ctx context.Context, id string) error {
```

Comment:
I think DisableKey would be better in to indicate what resource type this disables.
```suggestion
func (c *Client) DisableKey(ctx context.Context, id string) error {
```